src/main/resources/azure-common.properties

```diff
@@ -61,3 +61,11 @@ azure.apiVersionForStorageBlobService=2023-05-01
 azure.apiVersionForOpenShiftClusters=2023-04-01
 
 azure.armBased.vmSize.list="Standard_D2plds_v5","Standard_D4plds_v5","Standard_D8plds_v5","Standard_D16plds_v5","Standard_D32plds_v5","Standard_D48plds_v5","Standard_D64plds_v5","Standard_D2pls_v5","Standard_D4pls_v5","Standard_D8pls_v5","Standard_D16pls_v5","Standard_D32pls_v5","Standard_D48pls_v5","Standard_D64pls_v5","Standard_D2pds_v5","Standard_D4pds_v5","Standard_D8pds_v5","Standard_D16pds_v5","Standard_D32pds_v5","Standard_D48pds_v5","Standard_D64pds_v5","Standard_D2ps_v5","Standard_D4ps_v5","Standard_D8ps_v5","Standard_D16ps_v5","Standard_D32ps_v5","Standard_D48ps_v5","Standard_D64ps_v5","Standard_E2pds_v5","Standard_E4pds_v5","Standard_E8pds_v5","Standard_E16pds_v5","Standard_E20pds_v5","Standard_E32pds_v5","Standard_E2ps_v5","Standard_E4ps_v5","Standard_E8ps_v5","Standard_E16ps_v5","Standard_E20ps_v5","Standard_E32ps_v5","Standard_B2pls_v2","Standard_B2ps_v2","Standard_B2pts_v2","Standard_B4pls_v2","Standard_B4ps_v2","Standard_B8pls_v2","Standard_B8ps_v2","Standard_B16pls_v2","Standard_B16ps_v2","Standard_D2pls_v6","Standard_D4pls_v6","Standard_D8pls_v6","Standard_D16pls_v6","Standard_D32pls_v6","Standard_D48pls_v6","Standard_D64pls_v6","Standard_D96pls_v6","Standard_D2pds_v6","Standard_D4pds_v6","Standard_D8pds_v6","Standard_D16pds_v6","Standard_D32pds_v6","Standard_D48pds_v6","Standard_D64pds_v6","Standard_D96pds_v6","Standard_D2plds_v6","Standard_D4plds_v6","Standard_D8plds_v6","Standard_D16plds_v6","Standard_D32plds_v6","Standard_D48plds_v6","Standard_D64plds_v6","Standard_D96plds_v6","Standard_D2ps_v6","Standard_D4ps_v6","Standard_D8ps_v6","Standard_D16ps_v6","Standard_D32ps_v6","Standard_D48ps_v6","Standard_D64ps_v6","Standard_D96ps_v6","Standard_E2ps_v6","Standard_E4ps_v6","Standard_E8ps_v6","Standard_E16ps_v6","Standard_E32ps_v6","Standard_E48ps_v6","Standard_E64ps_v6","Standard_E96ps_v6","Standard_E2pds_v6","Standard_E4pds_v6","Standard_E8pds_v6","Standard_E16pds_v6","Standard_E32pds_v6","Standard_E48pds_v6","Standard_E64pds_v6","Standard_E96pds_v6"
+
+identifier.networkInterfaces=Microsoft.Network/networkInterfaces
+identifier.networkSecurityGroups=Microsoft.Network/networkSecurityGroups
+identifier.publicIPAddresses=Microsoft.Network/publicIPAddresses
+identifier.virtualNetworks=Microsoft.Network/virtualNetworks
+identifier.virtualMachines=Microsoft.Compute/virtualMachines
+identifier.virtualMachinesExtensions=Virtual machine extension
```

Comment:
`Virtual machine extension` is not a valid Azure resource type, pls use `Microsoft.Compute/virtualMachines/extensions` instead. See https://learn.microsoft.com/en-us/azure/templates/microsoft.compute/virtualmachines/extensions?pivots=deployment-language-bicep for more information.